File: 3 - Intro to Conditional Statements/Solution.ts
import readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const getline = () => {
  return new Promise(resolve => {
    rl.question("", resolve);
  });
};

(async () => {
  const N = parseInt((await getline()).toString());

  if (N % 2 !== 0) console.log("Weird");
  else if (N <= 5) console.log("Not Weird");
  else if (N <= 20) console.log("Weird");
  else console.log("Not Weird");

  rl.close();
})();

File: 3 - Intro to Conditional Statements/Solution.kt
fun main(args: Array<String>) {
    val N = readLine()!!.toInt()

    when {
        N % 2 != 0 -> println("Weird")
        N <= 5 -> println("Not Weird")
        N <= 20 -> println("Weird")
        else -> println("Not Weird")
    }
}
